src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -719,8 +721,20 @@ export default class OncoprintControls extends React.Component<
         }));
     }
 
-    @computed get filteredGenericAssayOptions() {
-        const allOptionsInSelectedProfile =
+    private isOptionIncludingText(text: string, option: ISelectOption) {
+        let result = false;
+        if (
+            !text ||
+            option.label.toLowerCase().includes(text.toLowerCase()) ||
```

Comment:
We should use case insensitive regexp to avoid all this casing stuff.  

(new RegExp(option.label,"i")).test(text)